bcda/models/models.go

```diff
@@ -63,7 +63,6 @@ type ACO struct {
 	ClientID           string       `json:"client_id"`
 	GroupID            string       `json:"group_id"`
 	SystemID           string       `json:"system_id"`
-	AlphaSecret        string       `json:"alpha_secret"`
 	PublicKey          string       `json:"public_key"`
```

Comment:
Can we also remove PublicKey? Look at it's usages, we only use it in cli.go (generates the PublicKey) but no one reads the column.